vsintegration/VisualFSharpVsix/RegisterFsharpPackage.pkgdef

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This is the manualy one. Currently, all VS assemblies are building their own pkgdef and inserting it into vsix. This file has the rest of the items that don't have a matching attribute yet.
I refactored what I could find, and will be refactoring some more with the roslyn changes.